global_phone.gemspec

```diff
@@ -14,4 +14,11 @@ Gem::Specification.new do |s|
   s.authors = ["Sam Stephenson"]
   s.email = ["sstephenson@gmail.com"]
   s.homepage = "https://github.com/sstephenson/global_phone"
+
+  if s.respond_to?(:metadata)
+    s.metadata['allowed_push_host'] = 'https://rubygems.pkg.github.com/callrail'
+    s.metadata["homepage_uri"] = s.homepage
+  else
+    raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
+  end
```

Comment:
we may also want to tweak some of the other values in this `gemspec`. I'm not sure if this would technically fall under the MIT license, and even if `Sam Stephenson` is the OG author, we probably don't want to link to his gmail in a corpo-repository

@moskeyombus do you have any thoughts about this?

Reply:
We can modify the the code under the MIT license, so the gemspec issues should be fine. However we have to retain the MIT license notice. MIT grants the right to sublicense which I believe means that we can add additional conditions on top of the existing license. I could be wrong about that though.